lib/apiservers/service/swagger.json

```diff
@@ -100,6 +100,47 @@
         }
       }
     },
+    "/target/{target}/vch/{vchId}/certificate": {
+      "get": {
+        "summary": "Access the host certificate for a VCH",
+        "description": "Making a GET request on /certificate under a VCH resource will return the certificate the VCH uses when acting as a server, which clients may wish to access to download and add to a trust store.",
+        "produces": [
+            "application/x-pem-file"
+        ],
+        "parameters": [
+          { "$ref": "#/parameters/target" },
+          { "$ref": "#/parameters/vch-id" },
+          { "$ref": "#/parameters/thumbprint" }
+        ],
+        "responses": {
+            "200": {
+                "$ref": "#/responses/vch-certificate"
+            },
+          "default": { "$ref": "#/responses/error" }
+        }
+      }
+    },
+    "/target/{target}/datacenter/{datacenter}/vch/{vchId}/certificate": {
```

Comment:
nit-pick: can you move this later in the document, so that it follows `/target/{target}/datacenter/{datacenter}/vch/{vchId}`?

Reply:
Good catch